src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -24,6 +24,8 @@ trait Ledger {
 
 class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with Logger {
 
+  val rewardTeller = new RewardTeller(blockchainConfig.monetaryPolicyConfig)
```

Comment:
If `object EVMConfig` was `class EvmConfigure(blockChainConfig)` then LedgerImpl could be `LedgerImpl(vm, rewardTeller)`. `blockChainConfig` really gets around. But this is outside the scope of this PR I think.